src/ddmd/s2ir.d

```diff
@@ -345,7 +345,8 @@ extern (C++) class S2irVisitor : Visitor
 
     override void visit(ForeachStatement s)
     {
-        printf("ForeachStatement.toIR() %s\n", s.toChars());
+        version (none)
```

Comment:
it's a debug print that should never show up.
also almost none of the other lowered constructs have a printf